skins/wikia/WikiaMonoBook.php

```diff
@@ -239,29 +239,4 @@ public function bottomScripts() {
 
 		return $html;
 	}
-
-	function wideSkyscraper() {
-		global $wgDBname;
-		$wideSkyscraperWikis = [ 'yugioh', 'transformers', 'swg', 'paragon' ];
-		if ( in_array( $wgDBname, $wideSkyscraperWikis ) ) {
-			echo ' style="margin-right: 165px;"';
-		}
-	}
-
-	function isSkyscraper() {
-		global $wgDBname, $wgEnableAdsInContent;
-		$noSkyscraperWikis = [ 'espokemon' ];
-		if ( in_array( $wgDBname, $noSkyscraperWikis ) && $wgEnableAdsInContent ) {
-			return true;
-		} else {
-			return false;
-		}
-	}
-
-	function noSkyscraper() {
-		if ( $this->isSkyscraper() ) {
-			echo ' style="margin-right: 0px;"';
-		}
-	}
-
 } // end of class
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[project-id]: wikia:app